staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -375,6 +375,11 @@ type CronJobSpec struct {
 	// The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
 	Schedule string `json:"schedule" protobuf:"bytes,1,opt,name=schedule"`
 
+	// The time zone for the given schedule, see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones.
+	// If not specified, this will rely on the time zone of the kube-controller-manager process.
+	// +optional
+	TimeZone string `json:"timeZone,omitempty" protobuf:"bytes,8,opt,name=timeZone"`
```

Comment:
this should be a pointer in all types.go files so we avoid serializing anything when the feature is off or the field is unset (would fix the unit test errors complaining about changing old API testdata on round trip)